views/layout.pug

```diff
@@ -0,0 +1,58 @@
+doctype html
+
+html
+	head
+		meta(charset="UTF-8")
+		meta(http-equiv="X-UA-Compatible" content="IE-edge, chrome=1")
+		meta(name="viewport" content="width=device-width, initial-scale=1")
+		title Styleguide Driven Development
+		link(rel="stylesheet", href="/css/core.0.12.0.css")
+		link(rel="stylesheet", href="/css/theme.css")
+	body
+		.svg-sprite(style="position: absolute; width: 0; height: 0;")
```

Comment:
This needs to also be a partial file in the includes folder that you include in the layout via the include statment. You also did not need to use pug you could have left the html as it was.